net-nntp/slrn/slrn-1.0.3.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_P="${P}a"
+
+DESCRIPTION="A s-lang based newsreader"
+HOMEPAGE="http://slrn.sourceforge.net/"
+SRC_URI="http://jedsoft.org/releases/${PN}/${MY_P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86 ~amd64-linux ~x86-linux"
+IUSE="canlock nls ssl uudeview"
+
+RDEPEND="virtual/mta
+	app-arch/sharutils
+	>=sys-libs/slang-2.2.3
+	canlock? ( net-libs/canlock )
+	ssl? ( dev-libs/openssl:0= )
```

Comment:
You don't need the 0 slot specifier, just `:=` should be fine.
Would you mind to add libressl support as well? You can find a description of the USE syntax for it [here](https://wiki.gentoo.org/wiki/Project:LibreSSL).